packages/konsole-app/src/panels/RecordPanel.tsx

```diff
@@ -49,20 +49,19 @@ const delay = 500;
 export const RecordPanel = () => {
   const classes = useStyles();
   const registryClient = useRegistryClient();
-  const [recordType, setRecordType] = useState<string>('');
+  const [recordType, setRecordType] = useState<string>('all');
   const [recordTypes, setRecordTypes] = useState<IRecordType[]>([]);
   const [records, setRecords] = useState<IRecord[]>([]);
   const [search, setSearch] = useState('');
   const [delayedSearch, setDelayedSearch] = useState(search);
+  const recordTypesWithAll = [{ type: 'all', label: 'all' }, ...recordTypes];
```

Comment:
Why is this required?

Reply:
'all' is needed in case user wants to see all the records, not just of specific types. Without this part, there would be no way to do that.  But I will think how to remove this 'all' constant

Reply:
Done